WpfSendKeys/SendKeysParser.cs

```diff
@@ -84,7 +33,10 @@ public IEnumerable<KeyPressInfo> ParseSendKeysString(string text)
                 }
                 else
                 {
-                    Error("Unexpected character: '" + CurrentChar + "'");
+					//TODO: HandleNewline here
+					ParseChar();
+
+                    //Error("Unexpected character: '" + CurrentChar + "'");
```

Comment:
We should check for `\r` and swallow it. Then we should have an `else` that errors if we encounter truly unexpected characters.